CHANGELOG.md

```diff
@@ -1,4 +1,23 @@
 # Changelog
+## Unreleased
+- Add ACMIDM and mocklogin to the stack [DL-6614]
+### Deploy instructions
+On DEV only: add the following to your `docker-compose.override.yml`:
+```
+  mocklogin:
+    image: lblod/mock-login-service:0.7.0
+    restart: "no"
+```
+And for QA and PROD:
+```
+  login:
+    environment:
+      MU_APPLICATION_AUTH_CLIENT_SECRET: "TODO_REPLACE_ME"
```

Comment:
Prod also needs different ACM/IDM variables, maybe we should add those to the docker-compose.yml file instead of the QA ones?

Reply:
Actually the main docker-compose file already has prod values, I can add the qa ones in the changelog though indeed